luigi/rpc.py

```diff
@@ -71,7 +72,7 @@ def _async_fetch(self, full_url, body):
 
     def _fetch(self, url, body, log_exceptions=True, attempts=3):
 
-        full_url = 'http://%s:%d%s' % (self._host, self._port, url)
+        full_url = 'http://%s:%d%s%s' % (self._host, self._port, self._url_prefix, url)
```

Comment:
Sorry, I should have read what you wrote more carefully. :)

Yea, the old % formatting is unpythonic and should be avoided.